.buildkite/pipeline.macos.yml

```diff
@@ -11,6 +11,8 @@ common: &common
     LANG: en_US.UTF-8
 
 prelude_commands: &prelude_commands |-
+  # Warm up ebs to avoid cold start
```

Comment:
Consider adding the stackoverflow link? Maybe we can do after we verified the Mac build passes for the first time (and in this way, we can retry one more time...)